pymc3/plots/kdeplot.py

```diff
@@ -3,15 +3,34 @@
 from scipy.signal import gaussian, convolve
 
 
-def kdeplot(trace_values, label=None, alpha=0.35, shade=False, ax=None,
-            **kwargs):
+def kdeplot(values, label=None, shade=0, ax=None, kwargs_shade={}, **kwargs):
+    """
+    1D KDE plot taking into account boundary conditions
+
+    Parameters
+    ----------
+    values : array-like
+        Values to plot
+    label : string
+        Text to include as part of the legend
+    shade : float
+        Alpha blending value for the shaded area under the curve, between 0
+        (no shade) and 1 (opaque). Defaults to 0
+    ax : matplotlib axes
+    kwargs_shade : dicts, optional
+        Additional keywords passed to `fill_between` (to control the shade)
```

Comment:
this is nitpicky, but can you replace `fill_between` with `matplotlib.axes.Axes.fill_between`?  I doubt there will be confusion, but specific is good.

Reply:
you are right, is better to be specific